Source/BusinessLogic/Logic/Players/PlayerRetriever.cs

```diff
@@ -147,6 +150,16 @@ internal virtual List<Champion> GetChampionedGames(int playerId)
                  .ToList();
         }
 
+        internal virtual List<GameDefinition> GetFormerChampionedGames(int playerId)
+        {
+            return
+                (from GameDefinition gameDefinition in
+                     dataContext.GetQueryable<GameDefinition>().Include(g => g.Champion)
+                 where gameDefinition.PreviousChampionId == playerId
```

Comment:
I see the problem. I should explain the "Champion" table better. Any time there is a change in Champion, a new Champion record is created. The Champion.Id is a technical key (identity). The PlayerId is the actual Player.Id of the Player who is was the champion. So instead of joining on playerId to a championId, you have to go through Champion.PlayerId.

Trying to fix this tonight but may run out of time as the baby is getting restless :P